### ISSUE TITLE
Should powershell and C# follow MS guidelines?

### ISSUE THREAD

**Issue by hpwaldegger** [SCS-CBU-CED-IAM/mobileid#51]
powershell and C# examples do not follow Microsoft guidelines e.g. for naming (cert_ca -> certCA, ...)

**Comment by phaupt** [comment]
I agree to follow best practice guidelines. Can you provide a reference / URL ?

In this Microsoft Powershell documentation, Microsoft provides examples with ```$my_var``` https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_variables?view=powershell-6
